src/Application/Commands/Command.php

```diff
@@ -4,30 +4,48 @@
 
 namespace MatchBot\Application\Commands;
 
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+use Psr\Http\Message\StreamInterface;
 use Symfony\Component\Console\Command\Command as SymfonyCommand;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 abstract class Command extends SymfonyCommand
 {
+    public const string CLI_OPTION_NOLOG = 'nolog';
+
     abstract protected function doExecute(InputInterface $input, OutputInterface $output): int;
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $this->start($output);
+        $this->start($input, $output);
         $return = $this->doExecute($input, $output);
-        $this->finish($output);
+        $this->finish($input, $output);
 
         return $return;
     }
 
-    protected function start(OutputInterface $output): void
+    protected function start(InputInterface $input, OutputInterface $output): void
     {
+        if ($this->noLog($input)) {
+            return;
+        }
+
         $output->writeln($this->getName() . ' starting!');
```

Comment:
I'm not sure if these "starting!" and "complete!" messages are ever useful, so I left them here in case but suppressed when the new `-l` / `--nolog` option is passed. I guess they would be useful in an environment where output from multiple command invocations is concatenated.

Reply:
@bdsl I think also useful in ECS where we have a log group that contains many task runs with alphanumeric IDs that don't mean anything on their own. It's definitely good to be able to try a few if you know the approximate time, or search Insights for the name from these logs.